Overview.src.html

```diff
@@ -336,17 +336,35 @@ <h4 id=terminology-headers>Headers</h4>
 <hr>
 
 <p>A <dfn>simple header</dfn> is a <span title=concept-header>header</span> whose
-<span title=concept-header-name>name</span> is `<code title>Accept</code>`,
-`<code title>Accept-Language</code>`, or `<code title>Content-Language</code>`, or whose
-<span title=concept-header-name>name</span> is `<code title>Content-Type</code>` and
-<span title=concept-header-value>value</span>, <span title=concept-header-parse>once parsed</span>,
-has a MIME type (ignoring parameters) that is
-`<code title>application/x-www-form-urlencoded</code>`, `<code title>multipart/form-data</code>`, or
-`<code title>text/plain</code>`.
+<span title=concept-header-name>name</span> is one of
+
+<ul class=brief>
+  <li>`<code title>Accept</code>`
+  <li>`<code title>Accept-Language</code>`
+  <li>`<code title>Content-Language</code>`
+  <li>`<code title>Content-Type</code>` and whose <span title=concept-header-value>value</span>,
+  <span title=concept-header-parse>once parsed</span>, has a MIME type (ignoring parameters)
+  that is `<code title>application/x-www-form-urlencoded</code>`,
+  `<code title>multipart/form-data</code>`, or `<code title>text/plain</code>`
+</ul>
+
 <!-- XXX * needs better xref
          * ignoring parameters has been the standard for a long time now
          * interesting test: "Content-Type: text/plain;" -->
 
+<p>or whose <span title=concept-header-name>name</span> is one of
+
+<ul class=brief>
+  <li>`<code title><a href=http://httpwg.org/http-extensions/client-hints.html#dpr>DPR</a></code>`,
+  `<code title><a href=http://httpwg.org/http-extensions/client-hints.html#downlink>Downlink</a></code>`,
+  `<code title><a href=http://httpwg.org/http-extensions/client-hints.html#save-data>Save-Data</a></code>`,
+  `<code title><a href=http://httpwg.org/http-extensions/client-hints.html#viewport-width>Viewport-Width</a></code>`, or
+  `<code title><a href=http://httpwg.org/http-extensions/client-hints.html#width>Width</a></code>`
```

Comment:
These should all have their own `<li>`.

Having said that, I would be okay with taking your text, modifying it a bit to match existing editorial style, and then committing it (with you still listed as author of course). The only downside of that is that this PR will be red, not purple. If that is a concern, I'll continue nitpicking a bit.